en/sql/index.rst

```diff
@@ -39,6 +39,8 @@ The main topics covered in this chapter are as follows:
 
 *   Database Administration: Describes about user management, SET and SHOW statements.
 
+*   User Schema: Describes a logical collection of objects in a database that is created with users.
```

Comment:
What's mean 'logical collection of objects'

Reply:
It means that objects that exist in the same schema do not exist in a physically separated space.

Objects in a schema are logically distinguished using the schema name as a prefix of the name. physically indistinguishable.

Reply:
I will add the explanation that they are not physically separated.